tasks_downstream.py

```diff
@@ -16,7 +16,11 @@
 from shutil import which
 
 from invoke import exceptions, task
-from invoke.util import yaml
+
+try:
+    import yaml
+except ImportError:
+    from invoke.util import yaml
```

Comment:
Why not using the Python standard `yaml` library always, then? Does it break something or introduce any constraints?

Reply:
Because if you install invoke with pipx, yaml won't exist inside its virtualenv unless you do the trick explained in https://github.com/Tecnativa/doodba-copier-template/pull/291#issue-1086681597. And it works for lower python versions. And in the future maybe upstream fixes https://github.com/pyinvoke/invoke/issues/820 and this doesn't become a problem when people start getting python 3.10 in their local machines.

Reply:
Oh I see, I thought it came as part of the standard Python libraries. Looks good then :+1: Just missing the CI fixes.